src/components/App/useInitTheme.ts

```diff
@@ -1,8 +1,17 @@
 import { useRecoilValue } from 'recoil';
 import { themeState } from 'state/preferences/themeAtom';
+import { setTheme } from '@ui5/webcomponents-base/dist/config/Theme';
 
 // you must initially load the themeState to
 //let the App know what theme to choose
 export const useInitTheme = () => {
-  void useRecoilValue(themeState);
+  const getCurrentTheme = () =>
+    window.matchMedia('(prefers-color-scheme: dark)').matches;
```

Comment:
maybe this would be a better naming choice and easier to read:
`const isSystemThemeDark = window.matchMedia('(prefers-color-scheme: dark)').matches;`